src/Messaging/NBB.Messaging.MultiTenancy/MessagingHeaders.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace NBB.Messaging.MultiTenancy
+{
+    public static class MessagingHeaders

```

Comment:
Should exist an extension method over Identification strategy builder that adds this default header.